src/components/HomepageFeatures/Hero.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+export default function Hero() {
+  return (
+    <section className="relative bg-slate-950 bg-hero-pattern bg-contain bg-no-repeat bg-bottom p-6 pb-0">
+      <div className="flex flex-col items-center justify-center mx-auto max-w-prose">
+        <p className="px-3 py-1 bg-slate-800 rounded-full z-10 lg:mb-3 mb-6">TCET Open Source</p>
```

Comment:
change `py-1` -> `pb-0.5` for more centered appearence
aligning text vertically is a bit of an hassle. this is accepted approach.

also add `text-gray-200`